cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -276,17 +276,44 @@ TEST(RayClusterModeTest, GetActorTest) {
   EXPECT_FALSE(ray::GetActor<Counter>("not_exist_actor"));
 }
 
-TEST(RayClusterModeTest, CreateAndRemovePlacementGroup) {
+ray::PlacementGroup CreateSimplePlacementGroup(const std::string &name) {
   std::vector<std::unordered_map<std::string, double>> bundles{{{"CPU", 1}}};
 
   ray::internal::PlacementGroupCreationOptions options{
       false, "first_placement_group", bundles, ray::internal::PlacementStrategy::PACK};
-  auto first_placement_group = ray::CreatePlacementGroup(options);
-  EXPECT_TRUE(ray::WaitPlacementGroupReady(first_placement_group.GetID(), 10));
+  return ray::CreatePlacementGroup(options);
+}
 
+TEST(RayClusterModeTest, CreateAndRemovePlacementGroup) {
+  auto first_placement_group = CreateSimplePlacementGroup("first_placement_group");
   ray::RemovePlacementGroup(first_placement_group.GetID());
 }
 
+TEST(RayClusterModeTest, CreateActorWithPlacementGroup) {
+  auto placement_group = CreateSimplePlacementGroup("first_placement_group");
+  EXPECT_TRUE(ray::WaitPlacementGroupReady(placement_group.GetID(), 10));
+
+  auto actor1 = ray::Actor(RAY_FUNC(Counter::FactoryCreate))
+                    .SetResources({{"CPU", 1.0}})
```

Comment:
If the required resources exceeds the resources of `PlacementGroup`, what will be happened? Can you add this test case?